apps/web/playwright/webhook.e2e.ts

```diff
@@ -422,23 +408,121 @@ test.describe("FORM_SUBMITTED", async () => {
     expect(page.locator(`text='${webhookReceiver.url}'`)).toBeDefined();
 
     await page.waitForLoadState("networkidle");
-    await page.goto("/routing-forms/forms");
-    await page.click('[data-testid="new-routing-form"]');
-    // Choose to create the Form for the user(which is the first option) and not the team
-    await page.click('[data-testid="option-0"]');
-    await page.fill("input[name]", "TEST FORM");
-    await page.click('[data-testid="add-form"]');
-    await page.waitForSelector('[data-testid="add-field"]');
-
-    const url = page.url();
-    const formId = new URL(url).pathname.split("/").at(-1);
-
-    await gotoRoutingLink({ page, formId: formId });
+
+    const form = await routingForms.create({
+      name: "Test Form",
+      userId: user.id,
+      teamId: null,
+      fields: [
+        {
+          type: "text",
+          label: "Name",
+          identifier: "name",
+          required: true,
+        },
+      ],
+    });
+
+    await gotoRoutingLink({ page, formId: form.id });
+    const fieldName = "name";
+    await page.fill(`[data-testid="form-field-${fieldName}"]`, "John Doe");
     page.click('button[type="submit"]');
 
     await waitFor(() => {
       expect(webhookReceiver.requestList.length).toBe(1);
     });
+    const [request] = webhookReceiver.requestList;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const body = request.body as any;
+    body.createdAt = dynamic;
+    expect(body).toEqual({
```

Comment:
Verify the payload as well, earlier were were just checking if the webhook was sent.